docs/sphinx/about/whats-new.txt

```diff
@@ -85,10 +108,6 @@ Other general improvements include:
 * fixes for the detection of csv pattern blocks by `FilePatternBlock`
```

Comment:
Should be double backticks here and in other cases in this file.

Also, the class names used throughout these notes should also be marked up with double backticks.  Double, NumberFormatException etc.